### ISSUE TITLE
🤖 - DaData input

### ISSUE THREAD

**Issue by splincode** [taiga-family/taiga-ui#3264]
### Description

https://dadata.ru/api/

# TODO

**Comment by vvgrad** [comment]
That sounds good
Could you describe in more detail what would you like to do?

**Comment by Intetix** [comment]
Interesting

**Comment by splincode** [comment]
@waterplea is it actual?

**Comment by waterplea** [comment]
Let's revisit it after we refactored all textfields

**Comment by splincode** [comment]
We ended up creating an experimental package for our own project:
https://github.com/taiga-family-labs/taiga-dadata

It provides a typed DaData integration for Angular, including suggestions for addresses, names, organizations, and banks, and can be used together with Taiga UI components.

The package is published as @taiga-ui-labs/dadata:
https://www.npmjs.com/package/@taiga-ui-labs/dadata

Demo:
https://taiga-family-labs.github.io/taiga-dadata/

It is still experimental, but if anyone needs similar functionality, feel free to use it, report issues, or contribute.

**Comment by waterplea** [comment]
Good idea, let's keep it a separate community package for the time being.
